Tools/Docusaurus/docs/Setup.md

```diff
@@ -3,15 +3,34 @@
 **Requirements**
 - [Unity 2019.4.31f1](https://unity3d.com/get-unity/download/archive)
 - [VRCSDK3 + Udon](https://vrchat.com/home/download)
-- [UdonSharp](https://github.com/Merlin-san/UdonSharp/releases)
 
 **Installation**
 
-> NOTE: This will be easier soon with the release of the VRChat Creator Companion. These notes are for the legacy ".unitypackage" format.
+You can get UdonSharp by using the [VRChat Creator Companion](https://vcc.docs.vrchat.com/) also known as the VCC, its [CLI](https://vcc.docs.vrchat.com/vpm/cli/), or a [starter template](https://github.com/vrchat-community/template-udonsharp). 
```

Comment:
```suggestion
You can get UdonSharp by using the [VRChat Creator Companion](https://vcc.docs.vrchat.com/) (also known as the VCC), its [CLI](https://vcc.docs.vrchat.com/vpm/cli/), or a [starter template](https://github.com/vrchat-community/template-udonsharp). 
```
This may help set the VCC abbreviation apart from the sentence after.